lib/cf-socket.h

```diff
@@ -28,6 +28,10 @@
 #include "nonblock.h" /* for curlx_nonblock(), formerly Curl_nonblock() */
 #include "sockaddr.h"
 
+#ifndef IPPROTO_MPTCP
+#define IPPROTO_MPTCP 262
+#endif
```

Comment:
maybe best to move this to `lib/cf-socket.c`, at the top (? or just above where you use it?), it is only used in this file, in a single place.